### ISSUE TITLE
grunt wiredep does not work after fresh install

### ISSUE THREAD

**Issue by michaeldewayneharris** [yeoman/generator-webapp_DEPRECATED#427]
grunt wiredep is documented in the information for webapp however it does not work after running "yo webapp"

in order to get it to work you need to:

1) npm install --save-dev grunt-wiredep (on console)
2)  add this to grunt.initConfig in gruntfile.js

```
   wiredep: {
      target: {

        // Point to the files that should be updated when
        // you run `grunt wiredep`
        src: [
          'app/*.html',   // .html support...
          'app/styles/*.scss'  // .scss & .sass support...
        ],

        // Optional:
        // ---------
        cwd: '',
        dependencies: true,
        devDependencies: false,
        exclude: [],
        fileTypes: {},
        ignorePath: '',
        overrides: {}
      }
    }
```

3) bower install --save jquery-ui
4) grunt wiredep (on console)

how to do this was gathered from https://github.com/stephenplusplus/grunt-wiredep

they also mention that you need to add  grunt.loadNpmTasks('grunt-wiredep'); to your grunt file but it did not seem to be required to add jquery-ui to various html documents.  I'm not sure what its for but it might be necessary for other automation

I would assume either the webapp documentation needs to be updated to include how to get wiredep setup or the generator-webapp needs to be modified to include wiredep by default.

**Comment by joined** [comment]
You have most probably and outdated version of `generator-webapp`.
In the past `grunt-bower-install` was used instead of `grunt-wiredep`, represented in Gruntfile with `bowerInstall` task.

The latest version of the generator uses `grunt-wiredep`, as you can see checking the [source](https://github.com/yeoman/generator-webapp/blob/master/app/templates/Gruntfile.js#L236-248).
To update to the latest generator version use `npm update -g generator-webapp`.

You mention that `grunt.loadNpmTasks` is not needed, this is because `load-grunt-tasks` plugin is used, which loads automatically the tasks.

**EDIT**: Actually, I just figured out that `grunt-wiredep` is used only from version `0.5.0-rc.1` that is not yet released on NPM, so you either stick with `0.4.9` which uses `grunt-bower-install` or you can install the latest release with `npm install -g generator-webapp@0.5.0-rc.1`.

**Comment by eddiemonge** [comment]
`npm install -g generator-webapp@0.5.0-2`
